src/components/Select/Select.tsx

```diff
@@ -127,38 +127,41 @@ export const Select: React.FC<SelectProps> = ({
   ]);
 
   return (
-    <div
-      id={`Select--${idRef}`}
-      className={classNames('Select', { 'Select--disabled': disabled }, className)}
-      ref={selectRef}
-      {...rest}
-    >
-      <Clickable
-        id={`Select__value--${idRef}`}
-        className='Select__value'
-        disabled={disabled}
-        onClick={handleToggle}
-        aria-haspopup='listbox'
-        aria-expanded={state.mode === Mode.Open}
-        aria-labelledby={`Select__label--${idRef} Select__value--${idRef}`}
-        type='button'
+    <div>
+      <input id={idRef} name={idRef} type='hidden' value={value} />
```

Comment:
I meant to put it inside the `div` below, not another wrapper div 😄 Sorry I wasn't clear enough on that.

Reply:
okay, got it!